companies/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+from authentication.models import CustomUser
+
+
+class Companies(models.Model):
+    company_id = models.BigAutoField(primary_key=True)
+    brand = models.CharField(max_length=255, blank=True)
+    is_registered = models.BooleanField(default=False)
```

Comment:
Got a question, we left this field? As iremember we renamed ir to the something like "is_startup"?